index.js

```diff
@@ -67,6 +69,15 @@ function fastifyWebsocket (fastify, opts, next) {
       callback(connection)
     })
   }
+  
+  fastify.addHook('onRequest', (request, reply, done) => {
+    if (request.raw[kWs]) {
+      request.ws = true
+    } else {
+      request.ws = false
+    }
+    done()
+  })
```

Comment:
Please add a test so future committers don't break this functionality!

Reply:
What test?

Reply:
I think it'd make sense to add unit tests that check the value of this property is true for a websocket route and false for a non websocket route.

Reply:
/@a-a-GiTHuB-a-a are you able to copy one of the existing unit tests to get this in?